package.json

```diff
@@ -1,23 +1,32 @@
 {
```

Comment:
NPM auto formats these, so it's a battle to keep original formatting; only significant change is I switched to `license` with a value of MIT